client.go

```diff
@@ -63,6 +63,8 @@ type Client struct {
 	// See http://www.sk-spell.sk.cx/tesseract-ocr-parameters-in-302-version
 	// TODO: Fix link to official page
 	ConfigFilePath string
+
+	isInit bool
```

Comment:
The name `isInit` doesn't make sense for me.
As I understand, when this flag is TRUE,

- re-set variables
- to already existing API object
- in order to update configuration

Is my understanding right?

Reply:
`isInit` is true when the instance is initialized (is the instance initialized => isInit)
So it should be as you describe when it is FALSE

Do you have a naming suggestion ? `shouldInit` and reverse the booleans ?

Reply:
According to your comment, I like it actually, yes, `shouldInit` looks better. It looks representing your comment more directly.